projects/plugins/jetpack/modules/theme-tools/featured-content.php

```diff
@@ -83,9 +83,12 @@ public static function setup() {
 		 * If no theme support is found there is no need to hook into WordPress. We'll
 		 * just return early instead.
 		 *
+		 * @deprecated $$next-version$$ Moved to Classic Theme Helper package.
+		 *
 		 * @uses Featured_Content::$max_posts
 		 */
 		public static function init() {
+			_deprecated_function( __METHOD__, 'jetpack-$$next-version$$', 'Automattic\\Jetpack\\Classic_Theme_Helper\\Featured_Content\\init' );
 			$theme_support = get_theme_support( 'featured-content' );
```

Comment:
On top of deprecating, should we load the matching methods from the package instead of keeping the code here?

Reply:
Sounds like a good idea. Thank you!